backend/src/DataSource/MqttSource.ts

```diff
@@ -57,12 +70,13 @@ export class MqttSource implements DataSource<MqttOptions> {
       ca: options.certificateAuthority ? Buffer.from(options.certificateAuthority, 'base64') : undefined,
       cert: options.clientCertificate ? Buffer.from(options.clientCertificate, 'base64') : undefined,
       key: options.clientKey ? Buffer.from(options.clientKey, 'base64') : undefined,
+      ...(options.psk ? pskOptions : {}),
     } as any)
 
     this.client = client
 
     client.on('error', (error: Error) => {
-      console.log(error)
+      console.log('client error', error)
```

Comment:
At some point, I think when I didn't uncheck the "Validate certificate" option, this was just logging 'true' over and over again.